var/spack/repos/builtin/packages/py-astroid/package.py

```diff
@@ -13,6 +13,7 @@ class PyAstroid(PythonPackage):
     homepage = "https://github.com/PyCQA/astroid"
     url      = "https://github.com/PyCQA/astroid/archive/astroid-1.4.5.tar.gz"
 
+    version('2.5.6', sha256='7fa655511c3726ce7058e9908065c385bcada47cb4cb31d5d6a7248b1fea58fa')
```

Comment:
Dependencies need to be updated, it looks like they are now documented in `setup.cfg`

Reply:
Hi @schmitts do you think you'll have a chance to address the comments here in the next few days?

Reply:
Yes, will do, @scheibelp.

Reply:
Done